core/src/main/java/de/danoeh/antennapod/core/service/download/DownloadServiceNotification.java

```diff
@@ -108,9 +108,11 @@ public void updateReport(List<DownloadStatus> reportQueue) {
             if (status.isSuccessful()) {
                 successfulDownloads++;
             } else if (!status.isCancelled()) {
-                createReport = true;
                 failedDownloads++;
             }
+            if (failedDownloads > 0 || showAutoDownloadReport && !status.isInitiatedByUser() && status.getFeedfileType() == FeedMedia.FEEDFILETYPE_FEEDMEDIA) {
```

Comment:
This is only about auto download. Please do not add something related to failed downloads here.

Reply:
Changed.